include/opendht/connstat.h

```diff
@@ -0,0 +1,97 @@
+/*
+ *  Copyright (C) 2014-2020 Savoir-faire Linux Inc.
+ *  Author(s) : Paymon <paymon@savoirfairelinux.com>
+ *
+ *  This program is free software; you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation; either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with this program. If not, see <https://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#include "log.h"
+
+#include <functional>
+#include <thread>
+
+struct nl_sock;
+struct nl_msg;
+
+namespace dht {
+namespace net {
+
+class OPENDHT_PUBLIC ConnectivityStatus
+{
+public:
+    enum class topic : const long unsigned int {
```

Comment:
`Topic` (Use CapitalizedCamelCase for type names)
`const long unsigned int` should not be required (if it is required, just use `unsigned long`)